### ISSUE TITLE
PolyPen: Unhandled Exception Caught!

### ISSUE THREAD

**Issue by garthhendy** [CGCookie/retopoflow#635]
Please tell us what you were trying to do, what you expected RetopoFlow to do, and what actually happened.
Provide as much information as you can so that we can reproduce the problem and fix it.
Screenshots and .blend files are very helpful.
Also, change the title of this bug report to something descriptive and helpful.
Thank you!

-------------------------------------

```
Environment:

- RetopoFlow: 2.0.0
- Blender: 2.79.0 master 2017-09-11
- Platform: Windows, 10, 10.0.17134, AMD64, AMD64 Family 23 Model 1 Stepping 1, AuthenticAMD
- GPU: NVIDIA Corporation, GeForce GTX 1080 Ti/PCIe/SSE2, 4.6.0 NVIDIA 411.63, 4.60 NVIDIA
- Timestamp: 2018-10-02 12:49:38.314414


Error Hash: 7b8b20170278afb7b60154fb5ba8160c


Trace:

- EXCEPTION (<class 'TypeError'>): glVertex2f() argument after * must be an iterable, not NoneType
- .../rfmode\rftool_polypen.py
- 000 0496:draw_lines() bgl.glVertex2f(*coords[0])
- 001 0576:draw_postpixel() self.draw_lines([bmv0.co, p0])
- .../common\profiler.py
- 002 0174:wrapper() return fn(*args, **kwargs)
- .../rfmode\rfcontext_drawing.py
- 003 0137:draw_postpixel() self.tool.draw_postpixel()
```

**Comment by vxlcoder** [comment]
Thanks for the report, @garthhendy!

This _should_ be fixed in commit 9aa5859.

**Comment by vxlcoder** [comment]
tested and could not reproduce.  I'm closing this issue, but feel free to reopen if it still happens.  Thanks!
